app/src/main/java/ca/rmen/android/poetassistant/Tts.java

```diff
@@ -128,30 +146,36 @@ public void onInit(int status) {
             }
             EventBus.getDefault().post(new OnTtsInitialized(status));
         }
-    }
+    };
 
     private final SharedPreferences.OnSharedPreferenceChangeListener mSettingsListener =
             new SharedPreferences.OnSharedPreferenceChangeListener() {
                 @Override
                 public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, String key) {
+                    if (!isReady()) return;
```

Comment:
If the Tts failed to initialize properly, we can't set the speech rate from the shared prefs right now.  But it's ok, the next time it's initialized, we'll do it then.